### ISSUE TITLE
Create and mantain protocol wiki

### ISSUE THREAD

**Issue by DStrelak** [I2PC/scipion#2017]
**Is your feature request related to a problem? Please describe.**
Sometimes it's not so clear which protocols can be used at which part of the pipeline.

**Describe the solution you'd like**
Ideally, there should be a wiki with all the protocols, including references to the (white) papers, example workflow using them, and some info on how to set the input and how to interpret output of the protocol.

**Comment by delarosatrevin** [comment]
This seems like a very broad and general issue...and not very actionable in a whole. Moreover, there are far more protocols on each plugin now. So the main documentation should come from the developers of each protocol.

**Comment by DStrelak** [comment]
I totally agree. But I believe that it is justifiable to require the authors to create documentation for their plugins, and update said docs with each new version (if applicable).
In the long term, it would be beneficial for the authors, as documented software is, generally speaking, more often used.

**Comment by azazellochg** [comment]
I think this issue was raised 4-5 years ago, good luck with that :rofl:
